File: csharpcore-Verify.xunit/GildedRoseFinal/SulfurasHandOfRagnarosItem.cs
namespace GildedRoseKata;

public class SulfurasHandOfRagnarosItem : Item
{
    public SulfurasHandOfRagnarosItem(int sellInDays, int quality) : base("Sulfuras, Hand of Ragnaros", sellInDays, quality)
    {
    }

    public override bool IsExpired()
    {
        return false;
    }

    public override void UpdateSellByDate()
    {
    }

    public override void UpdateQuality()
    {
    }
}

File: csharpcore-Verify.xunit/GildedRoseFinal/SulfurasHandOfRagnarosItem.java
package gildedrose.item;

import gildedrose.Item;

public class SulfurasHandOfRagnarosItem extends Item{

	public SulfurasHandOfRagnarosItem(int sellIn, int quality){
		super("Sulfuras, Hand of Ragnaros", sellIn, quality);
	}
	
	@Override
	public boolean isExpired(){
		return false;
	}

	@Override
	public void updateSellByDate(){
	}
	
	@Override
	public void updateQuality(){
	}
}
